modelreg/templates/registration/activation_complete.html

```diff
@@ -4,7 +4,8 @@
 {% block content-header %}{% trans "Activation complete" %}{% endblock %}
 {% block content-main %}
     {% blocktrans %}
-    Well done, thanks!
+    Thank you! Your account is now activated and you are able to
     {% endblocktrans %}
+    <a href="{% url "auth_login" %}">{% trans "Login" %}</a>.
```

Comment:
Please don't do this - it makes translation much, much harder, as this will generate two translation items that are not related to each other. Instead, include the <a> tag in the blocktrans.